odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewMetadata/MetadataItem/MetadataItemStyles.ts

```diff
@@ -58,4 +56,22 @@ export const Value = styled(Typography)<TypographyProps>(() => ({
   wordBreak: 'break-word',
   maxHeight: '5rem',
   overflow: 'auto',
+  '& > p': {
+    maxWidth: '200px',
+  },
+}));
+
+export const Pre = styled('pre')<{
+  $isOpened: boolean;
+}>(({ theme, $isOpened }) => ({
+  margin: 0,
+  whiteSpace: 'pre-wrap',
+  maxHeight: $isOpened ? 'none' : '60px',
+  overflowY: 'auto',
```

Comment:
I think you should delete Y overflow, because scroll bar looks weird.
<img width="930" alt="image" src="https://user-images.githubusercontent.com/59020701/154708749-b7055221-03d1-4906-b40a-b346e973a5bc.png">

Reply:
5 strings - done

Reply:
overflowY wasn't removed, but changed to hidden